README.md

```diff
@@ -474,6 +474,38 @@ services:
     restart: unless-stopped
 ```
 
+## Debugging containers
+
+Sometimes it is useful to debug the code which is running inside the container.  Two scripts (*'debug'* and *'debug_brk'* in the package.json file) are available to start NodeJs in debug mode, which means that NodeJs will start listening (to port 9229) for a debug client.
+
+1. In most cases the *'debug'* script will be sufficient, to debug a Node-RED application that is already running.  The NodeJs server can be started in debug mode using following command:
+   ```
+   docker run -it -p 1880:1880 -p 9229:9229 --name mynodered --entrypoint npm nodered/node-red run debug -- --userDir /data
+   ```
+
+2. In case debugging of the Node-RED startup code is required, the  *'debug_brk'* script will instruct NodeJs to break at the first statement of the Node-RED application.  The NodeJs server can be started in debug mode using following command:
+   ```
+   docker run -it -p 1880:1880 -p 9229:9229 --name mynodered --entrypoint npm nodered/node-red run debug_brk -- --userDir /data
+   ```
+   Note that in this case NodeJs will wait - at the first statement of the Node-RED application - until a debugger client connects...
```

Comment:
May be useful to also show/mention how you could start a debugger client - (or point off to some other docs).

Reply:
That indeed would be useful.  But in my case I use the Chrome debugger.  But others will most probably use Visual Studio Code, or whatever IDE...
If you want I can add a wiki page to this repository (and add a link to it from the readme page), where I explain the (few) steps that are required for the Chrome debugger?

Reply:
sure go ahead to add to wiki - and add a link to there from this readme.

Reply:
If you should have some time left, feel free too review it:
https://github.com/node-red/node-red-docker/wiki/Debug-container-via-Chrome-Developer-Tools